packages/@webex/plugin-meetings/src/meeting/index.ts

```diff
@@ -6907,84 +6907,73 @@ export default class Meeting extends StatelessWebexPlugin {
   }
 
   /**
-   * Publishes a track.
+   * Publishes a stream.
    *
-   * @param {LocalTrack} track to publish
+   * @param {LocalStream} stream to publish // TODO: this should probably be required...
+   * @param {MediaType} mediaType of the stream // TODO: ...and this should be required too
```

Comment:
A few TODOs here need to be addressed... I think we can make these both required and remove the check to see if `stream` exists also.